src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -279,9 +287,9 @@ public NativeInterface(AndroidJavaObject config)
       if (!isAttached) {
         AndroidJNI.AttachCurrentThread();
       }
-      using (var map = JavaMapFromDictionary(metadata))
+      using (AndroidJavaObject map = MakeJavaMapDisposable(metadata))
```

Comment:
nit: consider renaming to `MakeDisposableJavaMap` as the current naming implies that `metadata` is of type `java/util/Map`

Reply:
This is the same format as MakeJavaString so its consistent. Maybe Build is better than Make?

Reply:
Build sounds good to me